README.md

```diff
@@ -458,6 +458,41 @@ We use it for the control container because it needs to be available early to gi
 
 Be careful, and make sure you have a similar low-level use case before reaching for host containers.
 
+#### Bootstrap containers settings
+* `settings.bootstrap-containers.<name>.source`: The URI of the container
+* `settings.bootstrap-containers.<name>.mode`: The mode of the container, it could be one of `off`, `once` or `always`
+* `settings.bootstrap-containers.<name>.user-data`: field with arbitrary base64-encoded data
+
+Bootstrap containers are host containers that can be used to "bootstrap" the host before services like ecs-agent, kubelet and docker start.
+
+This type of containers are very similar to normal host containers, they come with persistent storage and with optional user data.
+Unlike normal host containers, bootstrap containers can't be treated as `superpowered` containers.
+However, these containers run as privileged and have access to the underlying root filesystem on `/.bottlerocket/rootfs`.
+Bootstrap containers are started once the `configured` systemd's target is reached, so they are not executed in a deterministic order.
+The boot process will "wait" for as long as the bootstrap containers run, and if any of them fail, the boot process will stop.
+
+Bootstrap containers have three different modes:
+
+* `always`: with this setting, the container is executed on every boot.
+* `off`: the container won't run
+* `once`: with this setting, the container only runs on the first boot. Upon completion, the container marks itself as `off`.
+If you need to execute the containers after the first boot, you can mark it as `once` and reboot.
+
+Here's an example of adding a bootstrap container with API calls:
+
+```
+apiclient -u /settings -X PATCH -d '{"bootstrap-containers": {"bootstrap": {"source": "MY-CONTAINER-URI", "mode": "once"}}}'
+apiclient -u /tx/commit_and_apply -X POST
```

Comment:
Can this example be written with `apiclient set`?  Or are multiple fields required here such that the simplified `set` mode is insufficient?

Reply:
I can write it with  `apiclient set --json` and specify the settings as a json blob or write multiple lines with `apiclient set bootstrap-containers.blah.*`

Reply:
Done :+1: